aws-lambda-java-tests/src/test/resources/msk_firehose_event.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Add a new test in https://github.com/aws/aws-lambda-java-libs/blob/main/aws-lambda-java-tests/src/main/java/com/amazonaws/services/lambda/runtime/tests/EventLoader.java

Reply:
added new tests in EventLoader.java and EventLoaderTest.java